release notes/v0.44.0.md

```diff
@@ -0,0 +1,220 @@
+webvital_largest_content_paint_good.......: 2       0.646663/s
+webvital_time_to_first_byte...............: avg=205.23ms min=104.79ms med=188.39ms max=322.5ms  p(90)=295.67ms p(95)=309.08ms
+webvital_time_to_first_byte_good..........: 3       0.969995/s
+```
+
+You may have noticed other metrics in there too. We rely on the [web-vitals](https://github.com/GoogleChrome/web-vitals) JS library which exposes a few more metrics, so we've left them in for you to experiment with.
+
+You will no longer see `browser_first_contentful_paint` in the summary, and instead you can work with `webvital_first_contentful_paint`.
+
+### UX improvements and enhancements
+
+- [#2906](https://github.com/grafana/k6/pull/2906). Added multiple date-time formats for CSV output. Thanks, @Azanul!
+- [#2916](https://github.com/grafana/k6/pull/2916). k6 started to show the actual binary's name in the usage help. Thanks, @ariasmn!
+- [#2942](https://github.com/grafana/k6/pull/2942). Reference 'k6 cloud' instead of 'Load Impact' in docs and errors.
+- [#2985](https://github.com/grafana/k6/pull/2985). Added support of async functions for `setup` and `handleSummary`.
+- [#2901](https://github.com/grafana/k6/pull/2901). Added a warning when the number of time series exceeds 100 000, which could lead to potential out-of-memory issues.
+- [#2997](https://github.com/grafana/k6/pull/2997). Added a new exit code (`109`), used on a go panic.
+- [xk6-browser#788](https://github.com/grafana/xk6-browser/pull/788) Updated the xk6-browser readme to highlight that it is now a module in k6.
+- [xk6-browser#803](https://github.com/grafana/xk6-browser/pull/803) Users are now warned if the `browser.close` method is called more than once.
+- [xk6-browser#820](https://github.com/grafana/xk6-browser/pull/820) Added error handling to wildcard selectors, which cleans up the error output in the terminal.
+- [xk6-browser#843](https://github.com/grafana/xk6-browser/pull/843) Removed the build step from github actions. From this release onwards, no new standalone xk6-browser binaries will be built and available from the [releases](https://github.com/grafana/xk6-browser/releases) section. The latest version of the browser module will be available in the k6 binary which can be found [here](https://github.com/grafana/k6/releases).
+- [xk6-browser#848](https://github.com/grafana/xk6-browser/pull/848) Multiple k6 instances can now connect to one browser to run concurrent tests. This update empowers high-concurrency browser testing with multiple VUs and instances. Using the new `browserType.Connect` API, users can now connect to an existing browser instance and execute concurrent tests, which was not possible previously.
+
+## Bug fixes
+
+- [#2984](https://github.com/grafana/k6/pull/2984). Fixed wrongly assigned HTTP `POST` and `PUT` methods for the `tracing.instrumentHTTP`. Thanks, @marcin-maciej-seweryn!
```

Comment:
```suggestion
- [#2984](https://github.com/grafana/k6/pull/2984) Fixed wrongly assigned HTTP `POST` and `PUT` methods for the `tracing.instrumentHTTP`. Thanks, @marcin-maciej-seweryn!
```
Same for all the following

Reply:
Fixed in [8642570](https://github.com/grafana/k6/pull/3024/commits/86425703a9724d53af4231acf0d6f081709410bf)